src/main/java/uk/nhs/hee/trainee/details/service/ProgrammeMembershipService.java

```diff
@@ -310,37 +299,15 @@ public boolean isPilot2024(String traineeTisId, String programmeMembershipId) {
     String managingDeanery = programmeMembership.getManagingDeanery();
     LocalDate startDate = programmeMembership.getStartDate();
     LocalDate dayBefore01082024 = LocalDate.of(2024, 7, 31);
-    LocalDate dayAfter31102024 = LocalDate.of(2024, 11, 1);
     if ((PILOT_2024_LOCAL_OFFICES_ALL_PROGRAMMES.stream()
         .anyMatch(lo -> lo.equalsIgnoreCase(managingDeanery)))
-        && (startDate.isAfter(dayBefore01082024) && startDate.isBefore(dayAfter31102024))) {
+        && (startDate.isAfter(dayBefore01082024))) {
```

Comment:
This needs updating to 2024-11-01

Reply:
I might be a bit confused, but I put some notes on this ticket: https://hee-tis.atlassian.net/browse/TIS21-6565